bin/subscription-manager-gui

```diff
@@ -22,6 +22,10 @@
 if __name__ != '__main__':
     raise ImportError("module cannot be imported")
 
+import sys
```

Comment:
An explanatory comment before this block would be helpful for folks coming back later.